src/maintainer/adding_pkgs.rst

```diff
@@ -111,6 +113,63 @@ The maintainer's job is to:
 - Answer eventual questions about the package on the feedstock issue tracker.
 
 
+Adding multiple packages at once
+--------------------------------
+
+If you would like to add more than one related packages, they can be added to 
+staged-recipes in a single pull request (in separate directories). If the 
+packages are interdependent (i.e. one package being added lists one or more of 
+the other packages being added as a requirement), conda-build will be able to 
+locate the dependencies that are only present within staged-recipes as long as 
+the builds finish in the dependencies order. Using a single pull request cuts 
+allows you to quickly get packages set up without waiting for each package in a 
+dependency chain to be reviewed, built, and added to the conda-forge channel 
+before starting the process over with the next recipe in the chain.
+
+.. note::
+
+   There will be a race condition if the packages are interdependent such that 
+   there will an error if a build finishes before its dependency is built.
+
+
+Adding additional packages in the future
+----------------------------------------
+
+If you would like to add additional packages in the future, you will need to reset your fork of staged-recipes before creating a new branch on your fork, adding the new package directory/recipe, and creating a pull request. This step ensures you have the most recent version of the tools and configuration files contained in the staged-recipes repository and makes the pull request much easier to review. The following steps will reset your fork of staged-recipes and should be executed from within a clone of your forked staged-recipes directory.
+
+#. Checkout your master branch
+
+  .. code-block:: none
+
+     git checkout master
+
+#. Define the conda-forge/staged-recipes repository as “upstream” (if you have not already done so).
+
+  .. code-block:: none
+
+     git remote add upstream https://github.com/conda-forge/staged-recipes.git
+
+#. Pull all of the upstream commits from the upstream master branch.
+
+  .. code-block:: none
+
+     git pull --rebase upstream master
```

Comment:
Is this step necessary if you reset --hard in the next step anyways?

Reply:
No, I forgot to remove it when I modified the steps to use rebase. I removed it.